java/common/src/main/java/org/apache/tsfile/enums/TSDataType.java

```diff
@@ -558,14 +579,19 @@ public boolean isComparable() {
         return true;
       case VECTOR:
       case BLOB:
+      case OBJECT:
         return false;
       default:
         throw new UnSupportedDataTypeException(this.toString());
     }
   }
 
   public boolean isBinary() {
-    return this == TEXT || this == STRING || this == BLOB;
+    return this == TEXT || this == STRING || this == BLOB || this == OBJECT;
+  }
+
+  public boolean isBlob() {
+    return this == BLOB || this == OBJECT;
```

Comment:
A little bit confusing? Actually, OBJECT is totaly different from BLOB in their storage. maybe we need to change the function name and add some java doc about what this metod is ued to?